tests/functional/spec/mci-refresh.spec.js

```diff
@@ -17,26 +16,10 @@ import ChangeCommentBlock from '../pages/surveys/mci-refresh/change-comment-bloc
 import TotalEmployees from '../pages/surveys/mci-refresh/total-employees.page.js'
 import EmployeesBreakdown from '../pages/surveys/mci-refresh/employees-breakdown.page.js'
 import Summary from '../pages/surveys/mci-refresh/summary.page.js'
-import devPage from '../pages/dev.page'
-import landingPage from '../pages/landing.page'
 import thankYou from '../pages/thank-you.page'
 
 describe('MCI 0215 Test', function() {
 
-  it('Given the mci business survey 0215 is selected when I start the survey then the landing page is displayed', function() {
-    // Given
-    devPage.open()
```

Comment:
Instead of removing this test you could call `openQuestionnaire('mci_refresh.json')`?